media/css/mozorg/home/fundraiser2017.scss

```diff
@@ -0,0 +1,206 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
+
+$amount-button-color: #81d4d0;
+$submit-button-color: #fef45f;
+
+#fundraiser-close {
+    background: url('/media/img/notification/close.svg');
+    border: none;
+    display: none;
+    height: 17px;
+    position: absolute;
+    right: 0;
+    top: -20px;
+    width: 17px;
+
+    // hide the 'Close' text
+    span {
+        @include visually-hidden;
+    }
+
+    // only display when JS is available
+    .js & {
+        display: block;
+    }
+}
+
+.fundraiser-fieldset {
+    margin-bottom: 20px;
+}
+
+#fundraiser {
+    background: #075f60 url('/media/img/home/fundraiser2017/background.png') center left no-repeat;
+    background-size: cover;
+    color: #fff;
+    padding: 40px 20px;
+    text-align: center;
+
+    // hide by default if JS is available to avoid flicker
```

Comment:
The only noticable downside to this is that there is a page jump when the donate section appears. It's not a big thing, but on slow connections (esp mobile) it could be a bit jarring maybe? This may be a little late design-wise, but it does make me wonder if it could be nicer to treat this like a fully-fledged section of the page, underneath the nav, and not like a notification or popup?

Reply:
You aren't wrong on any count above, including being late design-wise. ;)

I think it's better to be jarring coming in for users that haven't seen it yet than going out for users who have previously dismissed it.